### PR TITLE
Sensory intreface project: step 5.

### PR DESCRIPTION
**Pull request by mohammad-suliman**
This PR implements step 5 of the sensory interface project. Note also that it fixes a small bug with the "stopSoundPlayback" function, the bug had no effect on the functionality.

### CODE REVIEW

File: projects/sensory_interface/public/app.js [hunk 1]
```diff
@@ -131,4 +146,39 @@ function onCellChange(event) {
     let selectedValue = elementUnderTouch.firstChild.data;
     startSoundPlayback(selectedValue);
     event.stopPropagation();
+}
+
+function onRadioChange(radio) {
+    let minValuePicker = document.getElementById("minValue");
```
Comment (oferb) [inline projects/sensory_interface/public/app.js]: If values are already set, don't override them.
Example scenario:
User chooses manual, changes the values, then auto, then wants to go back to manual, but the values were overridden.

File: projects/sensory_interface/public/app.js [hunk 2]
```diff
@@ -131,4 +146,39 @@ function onCellChange(event) {
...
+        maxValuePicker.disabled = true;
+    } else {
+        minValuePicker.disabled = false;
+        maxValuePicker.disabled = false;
+    }
+    findMinAndMaxValues();
+}
+
+function findMinAndMaxValues() {
+    let input = document.getElementById("dataInput").value;
```
Comment (oferb) [inline projects/sensory_interface/public/app.js]: Please add a TODO to have a method to parse the string into a data structure (array of arrays, list of lists etc) and use it both here and in `processData()`.

**Comment by mohammad-suliman** [comment]
Thanks Ofer! A commit will come shortly to address the review comments.
I have a suggestion / consern:
Consider the following use case:
* The user opens the sensory interface project page.
* Then he chooses the "manual" option.
* Then he modifies the "min / max values" for example.
* Here is the issue: he must get back to the "process" button to show the table, the "update" button is still disabled in this stage.

So, do we need to refine this behavior? What do you think?

File: projects/sensory_interface/public/app.js [hunk 3]
```diff
@@ -66,6 +70,15 @@ function startSoundPlayback(selectedValue) {
     const MIN_FREQUENCY = 100;
     let minValue = document.getElementById("minValue").value;
     let maxValue = document.getElementById("maxValue").value;
+    maxValue = parseInt(maxValue);
```
Comment (oferb) [inline projects/sensory_interface/public/app.js]: The value is not necessarily an int. If can be any fractional number.

Comment (mohammad-suliman) [reply projects/sensory_interface/public/app.js]: No problem! Will fix that.

**Comment by mohammad-suliman** [comment]
What about making the "update" button enabled as soon as the user chooses the "manual" option? I think this is a better approach.

**Pull request merged**